flameshot.example.ini

```diff
@@ -48,6 +48,9 @@
 ;; Whether the tray icon is disabled (bool)
 ;disabledTrayIcon=false
 ;
+;; Use grim-based wayland universal screenshot adapter
+;useGrimAdapter=true
```

Comment:
Can you set this to =false to reflect the initial value in the code?

Reply:
Rebased on top of master branch and use `useGrimAdapter=false` in flameshot.example.ini